samples/ChatSample/Program.cs

```diff
@@ -19,6 +19,7 @@ public static void Main(string[] args)
                 .UseContentRoot(Directory.GetCurrentDirectory())
                 .UseIISIntegration()
                 .UseStartup<Startup>()
+                .UseKestrel().UseUrls("http://*.5000")//Add this Access to by IP
```

Comment:
I don't think we want to hardcode the port. Also, I believe the `.` is incorrect. I think It should be `*:5000` and not `*.5000`